js/apps/admin-ui/cypress.config.js

```diff
@@ -23,8 +24,10 @@ export default defineConfig({
     slowTestThreshold: 30000,
     specPattern: "cypress/e2e/**/*.{js,jsx,ts,tsx}",
     setupNodeEvents(on, config) {
-      on("after:spec", (spec, results) => {
-        if (results.video) {
+      // after:spec collides with cypressSplit function below and is overridden there
+
+      function afterSpecRemoveSuccessfulVideos(spec, results) {
```

Comment:
Since this function accepts all arguments it needs, it could be extracted into the root scope of the module. Also, it could use an early return on `results.video` to reduce nesting.